src/backend/Makefile

```diff
@@ -55,6 +55,7 @@ remote-db-migrations:
 		aspen-cli db --remote setup; \
 	fi
 	DB=remote alembic upgrade head
+	sleep 60
```

Comment:
@jgadling what do you think about this?

Reply:
I plan to remove that commit before I merge it in, but would be interested in hearing Jess' thoughts generally on the issue and why this fixes it. There was a Slack thread about it here:
https://chanzuckerbergteam.slack.com/archives/C01C310V70X/p1653587525484089